tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -315,14 +315,26 @@ def test_positive_install_sat_with_katello_certs(self, vm_setup):
         :CaseAutomation: Automated
         """
         cert_data, rhel_vm = vm_setup
+        version = str(settings.server.version.rhel_version)[0]
+        rhel_vm.download_repos(repo_name='satellite')
+        rhel_vm.register_contenthost(
+            org=None,
+            lce=None,
+            username=settings.subscription.rhn_username,
+            password=settings.subscription.rhn_password,
+        )
+        result = rhel_vm.subscription_manager_attach_pool([settings.subscription.rhn_poolid])[0]
+        for repo in (
+            f'rhel-{version}-server-extras-rpms',
```

Comment:
There's no extras channel in rhel8, default parametrization for contenthost will include rhel8. Do you want to limit this to only rhel7 testing?

Reply:
added support for RHEL8 as well.